autogpt_platform/backend/README.md

```diff
@@ -58,17 +58,18 @@ We use the Poetry to manage the dependencies. To set up the project, follow thes
 6. Migrate the database. Be careful because this deletes current data in the database.
 
    ```sh
-   docker compose up db redis -d
+   docker compose up db -d
    poetry run prisma migrate deploy
    ```
 
 ## Running The Server
 
 ### Starting the server without Docker
 
-Run the following command to build the dockerfiles:
+Run the following command to run database in docker but the application locally:
 
 ```sh
```

Comment:
Do you need the profile local if you do deps? Just checking, as I’m not super familiar

Reply:
The profile is added only to prevent the normal `docker compose up` to execute this `deps` service. Although `deps` pretty much does nothing, I just don't want this affecting the prod at all.